Sources/SnowManLogic/GameLogic/guessManager.swift

```diff
@@ -0,0 +1,64 @@
+public class GuessManager {
+	public var totalIncorrectGuessesAllowed: Int
+	public var correctGuesses: [String]
+	public var incorrectGuesses: [String]
+	public var guess: String
+
+	public init (totalIncorrectGuessesAllowed: Int) {
+		self.totalIncorrectGuessesAllowed = totalIncorrectGuessesAllowed
+		correctGuesses = []
+		incorrectGuesses = []
+		guess = ""
```

Comment:
I guess to clarify that point, anything in here that's using `guess` should just have guess passed into the method as an argument.